slices/fonts-urw-base35.yaml

```diff
@@ -0,0 +1,21 @@
+package: fonts-urw-base35
+
+essential:
+  - fonts-urw-base35_copyright
+
+slices:
+  config:
+    contents:
+      /etc/fonts/conf.d/61-urw*:
+      /usr/share/fontconfig/conf.avail/urw-*:
+
+  fonts:
+    contents:
```

Comment:
Should this include `_config` as an essential?

Reply:
Separation of the config and fonts had been requested specifically, so we probably don't need config unless we require it.